### ISSUE TITLE
oAuth2-Proxy fails to start when loading logo from URL

### ISSUE THREAD

**Issue by politician** [oauth2-proxy/oauth2-proxy#1407]
## Expected Behavior

oAuth2-Proxy should start and display my custom logo.

## Current Behavior

When using a URL as a `custom-sign-in-logo`, oAuth2-Proxy fails to start.

```
[2021/10/15 01:54:40] [main.go:60] ERROR: Failed to initialise OAuth2 Proxy: error initialising page writer: error loading logo: could not read logo file: open https://avatars.githubusercontent.com/u/62798169?s=200&v=4: no such file or directory
```

## Context

Here is the relevant part of my helm values file:

```yaml
extraArgs:
  custom-sign-in-logo: "https://avatars.githubusercontent.com/u/62798169?s=200&v=4"
```

## Your Environment

- Version used: 7.1.3 (via Helm chart 4.2.2)

**Comment by JoelSpeed** [comment]
This was fixed by #1330 and will be part of the 7.2.0 release

**Comment by Tantino** [comment]
add de variable:
```
            - name: OAUTH2_PROXY_CUSTOM_SIGN_IN_LOGO
              value: https://avatars.githubusercontent.com/u/62798169?s=200&v=4
```
